src/deploy/osp_deployer/settings/config.py

```diff
@@ -159,6 +159,40 @@ def __init__(self, settings_file):
         self.tenant_tunnel_vlanid = network_settings[
             'tenant_tunnel_network_vlanid']
         self.tenant_vlan_range = network_settings['tenant_vlan_range']
+        mtu_settings = self.get_settings_section(

```

Comment:
When the MTU variables are moved into the [Nodes Nics and Bonding Settings], most of the code will go away, and the remainder will move elsewhere.